src/main.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function daysInMonth(month, year) {
+  if (Number.isInteger(month) && Number.isInteger(year) && month < 13 && month !== 0) {
```

Comment:
you forgot about `daysInMonth(-1, 2019)`
```suggestion
  if (Number.isInteger(month) && Number.isInteger(year) && month < 13 && month > 0) {
```